Source/SPAppController.m

```diff
@@ -64,11 +64,14 @@ - (void)openSessionBundleAtPath:(NSString *)filePath;
 - (void)openColorThemeFileAtPath:(NSString *)filePath;
 - (void)openUserBundleAtPath:(NSString *)filePath;
 
+@property (readwrite, retain) NSFileManager *fm;
```

Comment:
I prefer full names... `fileManager`

Reply:
an easy change. There were already quite a few `fm`s so I just went with that.